.github/workflows/jit.yml

```
@@ -78,10 +78,10 @@ jobs:
             runner: macos-14
           - target: x86_64-unknown-linux-gnu/gcc
             architecture: x86_64
-            runner: ubuntu-22.04
+            runner: ubuntu-24.04
           - target: aarch64-unknown-linux-gnu/gcc
             architecture: aarch64
-            runner: ubuntu-22.04
+            runner: ${{ github.repository_owner == 'python' && 'ubuntu-24.04-aarch64' || 'ubuntu-24.04' }}
```

Maybe just add a comment here?

```suggestion
            # Forks don't have access to our paid AArch64 runners. These jobs are skipped below:
            runner: ${{ github.repository_owner == 'python' && 'ubuntu-24.04-aarch64' || 'ubuntu-24.04' }}
```